content/posts/hugo-blog-google-cloud-storage.md

```diff
@@ -0,0 +1,112 @@
++++
+title =  "Hosting a Hugo-Generated Blog on Google Cloud Storage"
+tags = ["hugo", "gcp"]
+date = "2024-07-22"
++++
+
+This site is hosted on Google Cloud using a Cloud Storage bucket to serve static HTML files generated using [Hugo](https://gohugo.io/).  I like this setup because it is _fast_, simple, and the content is easy to move to another platform or hosting provider if I want to.
+
+If that kind of setup appeals to you, this post can serve as a quick reference to get your own site up and running quickly.  I'm going to assume that you have already:
+
+* [Installed Hugo](https://gohugo.io/installation/) on your local computer
+* [Installed the `gcloud` CLI](https://cloud.google.com/sdk/docs/install-sdk) on your local computer, and logged in to your Google account
+
+
+<!-- # Test -->
```

Comment:
If I remember correctly, HTML comments will be carried through into generated HTML. To exclude this from the built output, use go template comments `{{/* comments here */}}`